core/fetcher/fetcher.go

```diff
@@ -39,6 +40,7 @@ type Fetcher struct {
 	awaitAttDataFunc func(ctx context.Context, slot, commIdx uint64) (*eth2p0.AttestationData, error)
 	builderEnabled   bool
 	graffitiBuilder  *GraffitiBuilder
+	electraSlot      eth2p0.Slot
```

Comment:
Note that this is currently not utilised. It will be once all VCs start asking. Then, if we are post electra, we can safely ask the BN only for 1 attestation data - with a committee index 0.

If there are strong objections not to keep it in the meantime, I'm OK with removing it.